src/connectors/components/CryptoExchangeMessageConnector.js

```diff
@@ -0,0 +1,75 @@
+// @flow
```

Comment:
This should be a "wired" component and its parent should decide where to get state from redux.